packages/react-server/src/ReactFlightServer.js

```diff
@@ -773,12 +773,8 @@ const Dispatcher: DispatcherType = {
     return callback;
   },
   useDebugValue(): void {},
-  useDeferredValue<T>(value: T): T {
-    return value;
-  },
-  useTransition(): [(callback: () => void) => void, boolean] {
-    return [() => {}, false];
-  },
+  useDeferredValue: (unsupportedHook: any),
+  useTransition: (unsupportedHook: any),
```

Comment:
Should we give heads up to @poteto for www?

Reply:
Yea